index.js

```diff
@@ -209,6 +210,7 @@ module.exports = {
           },
         ],
         "@typescript-eslint/no-continue": "off",
+		    "@typescript-eslint/consistent-type-imports": "error",
```

Comment:
This looks formatted funny, can you run prettier on it?

Reply:
I ran prettier on it, it reformatted the whole file